app/views/home/admin.html.haml

```diff
@@ -30,15 +30,14 @@
         %li= link_to 'Tags', admin_tags_path
         %li= link_to 'Tools', admin_tools_path
         %li= link_to 'Users', users_path
-
+  - if is_admin_or_project_admin_or_project_researcher
```

Comment:
A problem with this is that a user who is just an author won't see the Authoring link. 
Perhaps this change should be reverted and the whole section should be outdented.  That way it won't be inside of the 'if is_admin_or_manage' block.

I also notice that if statement on line 9 is only indented one space it should be indented 2 spaces. I think that also applies to the stuff below it.

Reply:
I made that change and it makes sense except that accounts that are only authors aren't allowed to access the admin page. Are they supposed to be?

Reply:
I'm not sure about authors seeing the admin page. I don't think there is a link to the authoring page otherwise. However the authoring page is kind of mess anyhow, so we probably don't want them to see it unless we fix it up.

I think the change is good, at least the logic matches the what seems to be expected. We might have to deal with author question for the Waters project so I think it is best to continue to punt it down the road until then.